k8s/nginx-ingress/prod-platform-ingress-resource.yaml

```diff
@@ -111,6 +111,16 @@ spec:
                 name: airqo-calibrate-app-svc
                 port: 
                   number: 80
+    - host: docs.airqo.net
```

Comment:
[docs custom subdomain](https://github.com/airqo-platform/AirQo-api/pull/1073/commits/81c31513896e39a498bb2bafe08cdc3541ea67d3)

Reply:
hi @123MwanjeMike. So we are hosting all the AirQo documentation (Engagements, partnerships, resources) on GitHub or just the AirQo API docs.

Reply:
Thanks @NoahNsimbe 
We are going to use it to host all the public information that includes the API and resources.
I am not sure if we shall also add the engagements and Partnerships, but @Baalmart can confirm.

Reply:
The Internal docs will remain in Postman as per now.

Reply:
engagements and partnerships can also reside here if we wish to. Just an effort to add version control to all our content. Thanks @123MwanjeMike for taking the initiative and acting on it! Amazing stuff! :)

Reply:
Great, thanks @Baalmart